utils/build/docker/php/apache-mod/entrypoint.sh

```diff
@@ -14,6 +14,26 @@ LOGS_APACHE=(/var/log/apache2/{access.log,error.log})
 touch "${LOGS_APACHE[@]}"
 chown root:adm "${LOGS_APACHE[@]}"
 
+# Fix the timeout
```

Comment:
This timeout fix is temporary, until we implement the ability to parse the time + unit value (tracked [here](https://github.com/DataDog/dd-appsec-php/issues/68)).